modules/core/include/opencv2/core/openvx/ovx_defs.hpp

```diff
@@ -24,6 +24,17 @@ namespace cv{
 namespace ovx{
 // Get common thread local OpenVX context
 CV_EXPORTS_W ivx::Context& getOpenVXContext();
+
+template <int kernel_id> inline bool skipSmallImages(int w, int h)     { return w*h < 3840 * 2160; }
```

Comment:
I think, this implementation-dependent checks must not be put to a header file; please, move them to .cpp's.